ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/rule-violations/index.tsx

```diff
@@ -68,9 +68,10 @@ import { paginationSchema, tableGroupingSchema } from '@/schemas';
 
 const defaultPageSize = 10;
 
-const columns: ColumnDef<RuleViolation>[] = [
-  {
-    accessorFn: (ruleViolation) => ruleViolation.severity,
+const columnHelper = createColumnHelper<RuleViolation>();
+
+const columns = [
+  columnHelper.accessor((ruleViolation) => ruleViolation.severity, {
```

Comment:
Does this need accessor function? `severity` is part of `RuleViolation` so wouldn't `id: 'severity'` be enough?